client/src/app/routes/chrysalis/Landing.tsx

```diff
@@ -291,7 +273,28 @@ class Landing extends Feeds<RouteComponentProps<LandingRouteProps>, LandingState
                                             )}
                                         </div>
                                     </div>
-
+                                    {isLatestMilesontFeedInfoEnabled && (
+                                        <div className="feed--metrics padding-l-8">
+                                            <div className="latest-index">
+                                                <h3>Latest Milestone:</h3>
+                                                <span className="metrics-value margin-l-s">
+                                                    {this.state.latestMilestoneIndex}
+                                                </span>
+                                            </div>
+                                            {this.state.secondsSinceLastMilestone !== undefined && (
+                                                <div className="seconds">
+                                                    <h3>Last & Target:</h3>
+                                                    <span className="metrics-value  margin-l-s">
+                                                        {this.state.secondsSinceLastMilestone}s / {
+                                                        this._networkConfig?.milestoneFrequencyTarget
+                                                        ? this._networkConfig.milestoneFrequencyTarget
+                                                        : 10
```

Comment:
Why do we have this random 10 default here ?

Reply:
If network config does not have `milestoneFrequencyTarget` field in it then `10` will be shown as a static milestoneFrequencyTarget value.